jdk12-reference/src/test/resources/spec-keys.yml

```diff
@@ -9,7 +9,26 @@ fieldPaths:
    path: $.nested.key
   target:
    path: $.nested.newKey
- - 
+ -
+  source:
+   path: $.someKey
+  target:
+   path: $.nested.someNewKey
+ -
+  source:
+   path: $.myString
+  target:
+   type: java.lang.Integer
+   path: $.myNumber
+# sourceIs allows the framework to use the newly created myNumber field (from the Target Object) as the source for the mappings.
+ -
+  sourceIs: target
```

Comment:
LGTM and i'll merge this but my thought is all source actions should be on the source object..

thoughts with that?